modules/tinymce/src/core/main/ts/keyboard/CommandUpDownKeys.ts

```diff
@@ -0,0 +1,77 @@
+
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Cell, Throttler } from '@ephox/katamari';
+import { PlatformDetection } from '@ephox/sand';
+import Editor from '../api/Editor';
+import { NodeChangeEvent } from '../api/EventTypes';
+import { EditorEvent } from '../api/PublicApi';
+import VK from '../api/util/VK';
+import * as InlineBoundariesNavigation from './InlineBoundariesNavigation';
+import * as MatchKeys from './MatchKeys';
+
+const platform = PlatformDetection.detect();
+
+const executeKeyupAction = (editor: Editor, evt: KeyboardEvent) => {
+  MatchKeys.execute([
+    { keyCode: VK.UP, metaKey: platform.os.isOSX(), action: MatchKeys.action(InlineBoundariesNavigation.moveToLineEndPoint, editor, false) },
+    { keyCode: VK.DOWN, metaKey: platform.os.isOSX(), action: MatchKeys.action(InlineBoundariesNavigation.moveToLineEndPoint, editor, true) }
+  ], evt);
+};
+
+const stopImmediatePropagation = (e: EditorEvent<NodeChangeEvent>) => e.stopImmediatePropagation();
+
+const isCommandUpDown = (evt: EditorEvent<KeyboardEvent>) =>
+  (evt.keyCode === VK.UP && evt.metaKey === true) ||
```

Comment:
This could be reduced to:
```
const isCommandUpDown = (evt: EditorEvent<KeyboardEvent>) => evt.metaKey && (evt.keyCode === VK.UP || evt.keyCode === VK.DOWN);
```

Reply:
this file was removed